### ISSUE TITLE
Single letter filename not supported

### ISSUE THREAD

**Issue by bendem** [fizzed/rocker#57]
`N.rocker.layout` will break rocker's maven plugin.

```
Failed to execute goal com.fizzed:rocker-maven-plugin:0.16.0:generate (generate-rocker-templates) on project server: Invalid template name format (unable find first dot character) -> [Help 1]
```

**Comment by bendem** [comment]
Guessing [this](https://github.com/fizzed/rocker/blob/3e7b7e17e5cbd09cd1fd6bbf2bb43fe824f9feb8/rocker-compiler/src/main/java/com/fizzed/rocker/compiler/RockerUtil.java#L129-L133) should be

```java
int pos = templateName.indexOf('.', 1);
// must be at least 1 char
if (pos < 0) {
    throw new IllegalArgumentException("Invalid template name format (unable find first dot character)");
}
```

**Comment by jjlauer** [comment]
Hi @bendem -- yeah I suppose that's possibly a good fix.  This would be
pretty simple to add a unit test for and verify if what you propose fixes
it.  Would be awesome if you could submit a PR for the fix!

**Comment by jjlauer** [comment]
Fixed in v0.17.0
